### PR TITLE
Admin portal base layout

### PR DESCRIPTION
**Pull request by brionmario**
## Purpose
"Admin portal" needs to be significantly different from the "User portal" and it needs more space on the screen.

## Goals
This PR brings a fluid layout design to the user portal which includes a side panel that supports child items.

## Approach
Side panel, header & footer components have been moved to the `@wso2is/react-components` and they support fluid and container layouts. Previous routing strategy has been changed and the portal now supports routes per each layout so that the router outlet can be placed anywhere on the layout to minimize unnecessary component re-renders.

Screen Shot 2020-01-30 at 5 10 40 AM

### CODE REVIEW

File: apps/admin-portal/src/components/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/components/index.ts]: This file was added last year I think we don't need to change the year here.

File: apps/admin-portal/src/app.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/app.tsx]: This file was added last year so we can keep the same license header I guess.

File: apps/admin-portal/src/components/index.ts (hunk 1 above)
Comment (brionmario) [reply apps/admin-portal/src/components/index.ts]: Fixed with comment `48590b7`

File: apps/admin-portal/src/app.tsx (hunk 2 above)
Comment (brionmario) [reply apps/admin-portal/src/app.tsx]: Fixed with commit `48590b7`

File: apps/admin-portal/src/components/side-panel.tsx [hunk 3]
```diff
@@ -1,186 +0,0 @@
...
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import * as React from "react";
```
Comment (DimalChandrasiri) [inline apps/admin-portal/src/components/side-panel.tsx]: Couldn't we separate the components into different files in order to make it more readable? WDYT?

Comment (brionmario) [reply apps/admin-portal/src/components/side-panel.tsx]: Actually this file was refactored and was moved to the new `@wso2is/react-components` module [1]. There, the components have been granularized. Please follow the bellow link, and let me know if you have any further change requests.

[1] https://github.com/wso2/identity-apps/tree/c1d5d905dc1b218ea6e07b00fe5b2b6c92719bc7/modules/react-components/src/side-panel

Comment (DimalChandrasiri) [reply apps/admin-portal/src/components/side-panel.tsx]: Looks good.

**Pull request merged**